src/Microsoft.PowerShell.Commands.Management/commands/management/WMIHelper.cs

```diff
@@ -162,7 +162,7 @@ internal override void StartOperation()
             }
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Verify PowerShell Runs.Verify Version 
```powershell
Expected regular expression '^7\.' to match $null, but it did not match.
at <ScriptBlock>, D:\_work\1\s\test\nanoserver\nanoserver.tests.ps1: line 16
16:         $version | Should -match '^7\.'
```